.github/workflows/ci.yml

```diff
@@ -0,0 +1,68 @@
+name: Continuous integration
+on: [push, pull_request]
+
+jobs:
+  # This matrix job runs the test suite against multiple Ruby versions
+  test_matrix:
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: [2.6, 2.7]
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    # Clone the govuk-content-schemas repo, a dependency of the govuk_schemas gem
+    # Make it available at ../govuk-content-schemas, which is where govuk_schemas expects it to be
+    - run: git clone -b deployed-to-production --single-branch https://github.com/alphagov/govuk-content-schemas ../govuk-content-schemas
+
+    # Gemfile.lock isn't needed when testing and releasing this gem – in fact, its existence causes problems.
```

Comment:
It'd be good to explain these problems in the commit. 

I could anticipate that this could cause problems as well since the contents of Gemfile.lock and gems installed could differ and cause test failures.